bin/fin

```diff
@@ -2415,6 +2433,12 @@ update ()
 	fi
 
 	[ $? -eq 0 ] && echo-green "Update finished"
+
+	if is_windows; then
+		echo-yellow "To use Docksal, start the console using the Docksal shortcut on your Desktop."
```

Comment:
@lmakarov I don't think we need this message here at all. As we discussed let's just silently install shortcut for now. 
And especially we don't need `read -p` here.